README.md

```diff
@@ -83,30 +126,52 @@ git submodule update --init --recursive
 set CMAKE_ARGS=-DONNX_USE_LITE_PROTO=ON
 pip install -e .
 ```
+### Linux
+
+First, you need to install protobuf. 
+
+Ubuntu users: the quickest way to install protobuf is to run
 
-### Building Protobuf from source
-* **Windows**
+```bash
+apt-get install python3-pip python3-dev libprotobuf-dev protobuf-compiler
```

Comment:
Shall we add comments to reminder users that `libprotobuf-dev` and `protobuf-compiler` from apt-get would be quite old protobuf versions since we do recommend to use protobuf 3.16.0?

Reply:
They are old but they should work without problems. I think we don't need to mention 3.16.0 specifically. Would you mind help me testing the command on Ubuntu 18.04? Supposedly it should work without problems.  You may need to add "cmake  git " to the apt-get install command line.

Reply:
I tested this updated document (`apt-get install python3-pip python3-dev libprotobuf-dev protobuf-compiler`) with Ubuntu 18.04 and the installation went well. Thanks for the update!